greenpithumb/pump.py

```diff
@@ -123,3 +129,16 @@ def is_running_pump_allowed(self):
                     return False
 
         return True
+
+    def pump_interval_exceeded(self):
+        """Returns True if plant needs to be watered now, False otherwise.
+
+        Plant needs to be watered regardless of soil moisture level if the time
+        elapsed since the last watering is greater than the pump interval.
+        """
```

Comment:
Suggest describing it in terms of the pump rather than the plant since the pump's behavior is independent of the plant.